packages/server/controllers/themeController.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import { CallbackError } from "mongoose";
+
+import { I_Theme, T_ThemeMode, Theme } from "../models/userModels";
+
+export class ThemeController {
+  create = async (req: Request<void, T_ThemeMode, I_Theme>, res: Response<string | CallbackError>) => {
+    try {
+      const { mode, userId } = req.body;
+      const dbTheme = await Theme.find({ userId });
+
+      if (dbTheme.length) {
+        void Theme.updateOne({ userId }, { mode }, {}, (err) => {
+          if (err) res.status(500).json(err);
+          res.json("ok");
```

Comment:
Сервер падает с ошибкой из-за отсутствия `else`
```
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
```
```suggestion
          else res.json("ok");
```